.eslintrc

```diff
@@ -2,116 +2,242 @@
     // eslint: recommended automatically enables most/all rules from the
     // possible errors section and more:
     // http://eslint.org/docs/rules/#possible-errors
-    "extends": "eslint:recommended",
+    "extends": ["plugin:prettier/recommended"],
```

Comment:
Should we move to eslint.yml?

Reply:
both json, js and yaml are supported, I migrated it to .js